build-w-docker.sh

```diff
@@ -13,6 +13,7 @@ cd $(dirname $0)
 export MSYS_NO_PATHCONV=1
 export MSYS2_ARG_CONV_EXCL="*"
 
+rm -rf dev
```

Comment:
I do not understand how could it affect. Could you give us some details?

Reply:
This is because we compare the compilation generated in the dev folder, and it is necessary to eliminate it.